Core/GameEngine/Source/Common/Audio/AudioEventRTS.cpp

```diff
@@ -800,23 +800,22 @@ AsciiString AudioEventRTS::generateFilenameExtension( AudioType audioTypeToPlay
 //-------------------------------------------------------------------------------------------------
 void AudioEventRTS::adjustForLocalization(AsciiString &strToAdjust)
 {
-	const char *str = strToAdjust.reverseFind('\\');
-	if (!str) {
-		return;
+	const char *filename = strToAdjust.reverseFind('\\');
+	if (filename) {
+		filename += 1;
```

Comment:
Skips the `\`. The path returned by `generateFilenamePrefix` already has trailing `\`